### PR TITLE
Add initial repo definitions for new prometheus exporters & team for them

### PR DESCRIPTION
**Pull request by jacobsee**
Repos for some of our infra monitoring prometheus exporters.

Questions:
1. Should `pushbot` be on these at all if we're going to use Quay's built-in image builder?
2. Should the team name be `prometheusexporters` or something like `prometheusexportermaintainers`? Seemed to be getting a bit long...

cc: @oybed

### CODE REVIEW

**Comment by jacobsee** [comment]
And I suppose a third question: Should anyone else be on the prometheus exporter maintainer team?

**Comment by oybed** [comment]
@jacobsee A couple of suggestions:
1. pushbot - no, not needed and should be fine without, but also doesn't hurt having one set up for future use. I'll leave it up to you. 
2. team name - I'd probably just recommend that we name it the same as the repo - i.e: `monitoring` 
3. With the lack of grouping in quay.io, how about naming the images with a prefix that make them easier to identify - perhaps something like `monitoring-` or `prometheus-` ... it will make for a somewhat long name, but IMHO that's ok.
4. team members - please add myself for now and we can always update later as more people get involved

**Comment by jacobsee** [comment]
@oybed Sounds good on all counts 👍

**Review by oybed — APPROVED** [review]
/lgtm

**Pull request merged**
